src/App/tests/App.test.tsx

```diff
@@ -64,6 +68,19 @@ describe('App', () => {
     wrapper.unmount();
   });
 
+  it('tracks pages correctly', () => {
+    GoogleAnalytics.pageview = jest.fn();
+    const wrapper = mount(
+      <Provider store={store}>
+        <Router history={history}>
+          <App />
+        </Router>
+      </Provider>
+    );
+    expect(GoogleAnalytics.pageview).toBeCalledWith('/');
+    wrapper.unmount();
+  });
+
```

Comment:
@kelvin-muchiri can we navigate to some other pages and check that those pages were tracked okay?  I suppose that we'd also want to check that the username was tracked okay.

Reply:
@moshthepitt Yea sure. I am having trouble navigating to other pages or changing the value of the envs at the moment.

Reply:
@kelvin-muchiri have you tried using the pushing on the history object?
```
import { createBrowserHistory } from 'history';
const history = createBrowserHistory();
...
history.push(PLAN_LIST_URL);
```

@moshthepitt @p-netm - is this the best way to navigate?

Reply:
@cKellyDesign @kelvin-muchiri I would first try to find the links in the enzyme wrapper and then click on them to go to a different page.   But it is also acceptable to do history.push

Reply:
> I would first try to find the links in the enzyme wrapper and then click on them to go to a different page

Good call, this more closely resembles user interaction. Since we also have places which push history, maybe we should test both cases.

Reply:
@moshthepitt @cKellyDesign Should we really worry about the implementation in this case?

Reply:
I actually realized other tests made the tests fail. Commenting out other tests lead to the tests passing

Reply:
@kelvin-muchiri Commenting out?  That sounds .... weird :)

Reply:
@moshthepitt I think it was to do with the history. Resetting the history before each test has fixed it.

Reply:
> Commenting out? That sounds .... weird :)

@kelvin-muchiri I think this means the commented out code should be removed from the file and committed so that current git history does not include commented out tests. 😄

Reply:
@cKellyDesign  Yea sure